pkg/module/http2/transport.go

```diff
@@ -2604,3 +2604,20 @@ func traceFirstResponseByte(trace *httptrace.ClientTrace) {
 		trace.GotFirstResponseByte()
 	}
 }
+
+func traceHasWroteHeaderField(trace *httptrace.ClientTrace) bool {
```

Comment:
why delete go111.go and not_go111.go? seems that we should still keep these files? 
ref: https://cs.opensource.google/go/x/net/+/master:http2/go111.go

Reply:
do you want to compile in a version less than go 1.11? I deleted <1.11 support which those files were required for...

Reply:
I left removing this for another person. It isn't critical to do in this PR